editor/store/effects.js

```diff
@@ -472,7 +472,7 @@ export default {
 	},
 	INITIALIZE_META_BOX_STATE( action, store ) {
 		// Allow toggling metaboxes panels
-		if ( some( action.metaBoxes ) ) {
```

Comment:
It would be great if we could move all the meta boxes code to a hook, all these special cases will be handled more elegantly.

Reply:
Related https://github.com/WordPress/gutenberg/pull/5175 I've moved the meta boxes to the `edit-post` module, so if you build your own UI to the editor, you won't have to deal with meta boxes anymore